shell/platform/darwin/macos/framework/Source/FlutterViewControllerTest.mm

```diff
@@ -1004,6 +1011,39 @@ - (bool)testTrackpadGesturesAreSentToFramework:(id)engineMock {
   return true;
 }
 
+- (bool)mouseAndGestureEventsAreHandledSeparately:(id)engineMock {
```

Comment:
Optional: Might be worth adding a comment mentioning Apple's Magic Mouse.

Reply:
It is mentioned inside the test but that's probably too easy to overlook.

Reply:
Ah yes -- somehow glossed over that on line 1034!